sdk/keyvault/azkeys/client_test.go

```diff
@@ -581,9 +579,8 @@ func TestImportKey(t *testing.T) {
 			client, err := createClient(t, testType)
 			require.NoError(t, err)
 
-			r := RSA
 			jwk := JSONWebKey{
-				KeyType: &r,
+				KeyType: KeyTypeRSA.ToPtr(),
```

Comment:
Interesting that this is a pointer. It is a required field of a JWK, or are there cases - either in the swagger or in the SDK - where it could be `nil`?

Reply:
This is in response types which are empty when an error occurs. We want to have it as nil for this scenario where there's a difference between "not there because the operation failed" and "zero value"